test/test_schedule.py

```diff
@@ -734,6 +834,19 @@ def test_reduce_simple_chase(self):
     schedule = check_schedule([b, c], 3)
     assert schedule[0].ast[0].src[0].op is BinaryOps.ADD
 
+  # multireduce spec
+  @unittest.skip
+  def test_multireduce_simple_chase(self):
+    a = Tensor.rand(4, 4, 4)
+    r = (a + (a.sum(0, keepdim=True) + 6)).sum(0) * 2
+    b = r.sum(0) + 8
+    c = r.sum(1) + 12
+    np_r = (a.numpy() + (a.numpy().sum(0) + 6)).sum(0) * 2
+    np.testing.assert_allclose(b.numpy(), np_r.sum(0) + 8)
+    np.testing.assert_allclose(c.numpy(), np_r.sum(1) + 12)
+    schedule = check_schedule([b, c], 3)
```

Comment:
you can't reschedule b and c.
these should be `s = create_schedule()` then `run_schedule(s)`